packages/core/admin/package.json

```diff
@@ -72,7 +72,6 @@
     "esbuild-loader": "^2.20.0",
     "execa": "^1.0.0",
     "fast-deep-equal": "3.1.3",
-    "find-root": "1.1.0",
```

Comment:
can remove it from here and yarn then

Reply:
That is it being removed :) I think another package in the repo needs it hence why it still shows in the `yarn.lock`.